src/device-registry/bin/www

```diff
@@ -196,7 +227,12 @@ cluster(function() {
             run().catch(error => {
                 logger.error(`KAFKA: internal server error -- ${JSON.stringify(error)}`);
             });
+
+            emitLatestMeasurements().catch(error => {
```

Comment:
@BenjaminSsempala , to test this configuration, we need to create a small frontend application in either Flutter or React and then proceed accordingly.

Thanks!